gradle/libs.versions.toml

```diff
@@ -1,89 +1,90 @@
 [versions]
-assetModel = "0.1.2"
-assetSpec = "1.0.0"
-bouncyCastle = "1.70"
+asset-model = "0.1.2"
+asset-specs = "1.0.0"
+blockapi = "0.1.3"
+bouncycastle = "1.70"
 feign = "11.8"
+figure-eventstream = "0.7.1"
 grpc = "1.47.0"
 jackson = "2.13.3"
-jacksonProtobuf = "0.9.12"
+jackson-protobuf = "0.9.12"
 kotlin = "1.6.21"
-kotlinCoroutines = "1.6.2"
-kotlinLogging = "2.1.21"
+kotlin-coroutines = "1.6.2"
+kotlin-logging = "2.1.21"
 logback = "1.2.6"
 mockk = "1.12.4"
-okHttp = "4.9.3"
+objectstore-gateway = "3.1.2"
+okhttp = "4.9.3"
 protobuf = "3.21.1"
-provenanceClient = "1.1.1"
-provenanceEventStream = "0.6.3"
-provenanceGateway = "2.0.2"
-provenanceHdWallet = "0.1.15"
-provenanceProto = "1.10.0"
-provenanceScope = "0.6.0"
+provenance-client = "1.3.0"
```

Comment:
Real change: Upgraded to v1.3.0 for PbClient to access the groups goodness in downstream consumers without having to re-import pbClient